backend/tests/db/user.test.ts

```diff
@@ -288,4 +289,30 @@ describe("db/models/user", () => {
             selected: false,
         } satisfies APIv4.UserSectionAttrs);
     });
+
+    test("replace capital letters with lowercase in saml", async () => {
+        const uid1 = await getOrCreateUser("First Test User", "");
+        const uid2 = await getOrCreateUser("Second Test User", "");
+        const uid3 = await getOrCreateUser("third test user", "");
+        const issuerId = await getOrCreateUser(
+            "JBao00@cmc.edu",
```

Comment:
Try not to use the name of a real student for test data

Reply:
Got it. I adjusted the test case.